confluence/confluence-v1.asciidoc

```diff
@@ -0,0 +1,73 @@
+Confluence Rest Configuration
+----------------------------
+This documentation describs what data the confluence configuration retrives and the endpoints used.
+This is this list of data that rest connector using confluence configuration will crawl.
+
+* spaces 
```

Comment:
In the datasource, I don't see the 'spaces' endpoint configured  `/wiki/rest/api/space` , does it need to be added now?